manage.py

```diff
@@ -13,14 +18,18 @@ def distribute():
     You need: PyPI username and password.
     You need to type in the PyPI password rather than copy-pasting.
     """
-
+    create_venv()
     install()
 
     os.system(activate() + ' && ' + 'pip install --upgrade build')
     os.system(activate() + ' && ' + 'python -m build')
     os.system(activate() + ' && ' + 'pip install --upgrade twine')
     os.system(activate() + ' && ' + 'twine upload dist/*')
 
+def create_venv():
```

Comment:
Add a snippet here that does nothing if you already have an existing environment please.

Reply:
I modified it.